dsp-user/core/src/test/scala/dsp/user/sharedtestdata/SharedTestData.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package dsp.user.sharedtestdata
+
+import dsp.valueobjects.User._
+import dsp.user.domain.User
+
+object SharedTestData {
+  val simpleGivenName1  = GivenName.make("GivenName1").fold(e => throw e.head, v => v)
```

Comment:
Should we ever throw? I thought that a `make` returns a `Validation`?

Reply:
It does, but then we either return a value object or throw an error. At least this is how it's now implemented.

Reply:
I hope we cleared this part yesterday (no throwing allowed in ZIO or you need to wrap it inside a `ZIO.attempt`)